src/deadline_worker_agent/sessions/job_entities/job_details.py

```diff
@@ -91,33 +91,39 @@ def job_run_as_user_api_model_to_worker_agent(
     """Converts the 'JobRunAsUser' api model to the 'JobRunAsUser' dataclass
     expected by the Worker Agent.
     """
-    if "runAs" in job_run_as_user_data and job_run_as_user_data["runAs"] == "WORKER_AGENT_USER":
+    # Only two options for "runAs": WORKER_AGENT_USER & QUEUE_CONFIGURED_USER
+    if job_run_as_user_data["runAs"] == "WORKER_AGENT_USER":
```

Comment:
`runAs` is required now, so I believe we have to update the api_model file to indicate as such? it's currently marked as `NotRequired`

Reply:
Good catch. I'll update that.